axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
```

Comment:
Very good point, I'd still say too many tests is better than too phew but I won't insist. :) :+1: